### PR TITLE
Fuck it, we ball: Makes all abnormality lists look the same, makes github code changes in lists look better, replaces MORE FUCKIN SPANS and some more stuff!

### PR DESCRIPTION
**Pull request by Gboster-0**
## About The Pull Request

replaces lists like this:
list( something1, something2, something3)
and
list(something1,
something2,
something3)
with
list(
    something1,
    something2,
    something3,
)

Additionally i made comments on foresaken murderer into proper codeblocks, to make it more readable
And replaced the " ' " symbols into " /" " inside of chat messages, as that looks clearer
...And makes hardcore rolling not pick a pick. Because im quite sure its not supposed to pick a pick
...Also it replaces the " ’ " symbol with " ' " inside of text, because the first symbol highlights yellow in-code for no reason, so i think its better to not use it
...Oh yeah, also i added 	/mob/living/simple_animal/hostile/abnormality/pinocchio = 1.5 to OZ abnormalities, as they had it commented out from before pinocchio existed and nobody ever noticed. Fun how that works huh?

## Why It's Good For The Game

<details>
<summary>How were github code changes looking changed on lists?</summary>

Basically, if you added "something 3" to a list it would look like this in file preview on github:

somelist(
something,
-something2)
+something2,
+something 3)

Instead with proper lists it will show:

somelist(
something,
something2,
+something3,
)

Making things clearer

</details>

## Changelog
:cl:
spellcheck: fixed a few typos
code: lists were made universally the same, and now show changes more accuratelly on github's file preview
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //ho ho hoe -gail
 /mob/living/simple_animal/hostile/abnormality/rudolta
 	name = "Rudolta of the Sleigh"
-	desc = "An abnormality consisting of three parts: A hornless, disfigured reindeer, 'Santa' and a sleigh. \
+	desc = "An abnormality consisting of three parts: A hornless, disfigured reindeer, \"Santa\" and a sleigh. \
```
Comment (InsightfulParasite) [inline code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm:4]: Is that how you do "" in html text? i assumed it was impossible.

File: code/modules/mob/living/simple_animal/abnormality/he/will_you_play.dm [hunk 2]
```diff
@@ -71,9 +71,9 @@
 /mob/living/simple_animal/hostile/abnormality/willyouplay/proc/Tie(mob/living/carbon/human/user, work_type)
 	if(janken == 0)
 		SLEEP_CHECK_DEATH(20)
-		say("A draw. Did you think I wouldn�t play scissors?")
+		say("A draw. Did you think I wouldn�t play scissors?")
 		SLEEP_CHECK_DEATH(20)
-		say("I don�t play with folks who don't trust me.")
+		say("I don�t play with folks who don't trust me.")
```
Comment (InsightfulParasite) [inline code/modules/mob/living/simple_animal/abnormality/he/will_you_play.dm:76]: This is out of my jurisdiction but due to how big the PR is ill highlight it. I have no clue what the ? is.

File: code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm (hunk 1 above)
Comment (Gboster-0) [reply code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm:4]: yeah the " \ " character will make any character thats in front of it automatically count as a normal letter instead of a special one.
Thats how you get "" inside of descriptions

File: code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm [hunk 3]
```diff
@@ -8,10 +8,14 @@
 	icon_dead = "forsakenmurdererdead"
 	portrait = "forsaken_murderer"
 	del_on_death = FALSE
-	/* Originally was 270. Fragment health is 800 with a original game health of 230 so techically forsaken
-		murderer has more health than fragment? Ill round the numbers to 600 since 270 can be rounded to 300 and doubled.
-		I was later told to make it 1100. We really dont have a Lobotomy Corp to LC13 health conversion calculator.*/
-	// Turns out that 1100 is not enough. - Kirie/Kitsunemitsu
+	/**
+	 * Originally was 270.
```
Comment (InsightfulParasite) [inline code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm:12]: As the coder for Forsaken i do approve of this reformatting of the comments to make them more readable.

File: code/modules/mob/living/simple_animal/abnormality/he/will_you_play.dm (hunk 2 above)
Comment (Gboster-0) [reply code/modules/mob/living/simple_animal/abnormality/he/will_you_play.dm]: yeah uh, honesly no clue how that happened. Thats supposed to be the " ' " character

Comment (InsightfulParasite) [reply code/modules/mob/living/simple_animal/abnormality/he/will_you_play.dm]: Gotcha!

**Comment by InsightfulParasite** [comment]
Looks good to me. Its alot of the same changes among many files with span being changed to its macro and code being moved around for readability. Im not a maintainer but thats what i got from sweeping over the changes.

**Review by EgorDinamit — DISMISSED** [review]

File: code/game/machinery/computer/abnormality_queue.dm [hunk 4]
```diff
@@ -81,7 +81,7 @@
 			UpdateAnomaly(target_type, "fucked it lets rolled", TRUE)
 		if("hardcore_fuck_it_lets_roll")
 			if (SSabnormality_queue.hardcore_roll_enabled)
-				var/mob/living/simple_animal/hostile/abnormality/target_type = pick(pick(SSabnormality_queue.possible_abnormalities))
+				var/mob/living/simple_animal/hostile/abnormality/target_type = pick(SSabnormality_queue.possible_abnormalities)
```
Comment (LanceSmites328) [inline code/game/machinery/computer/abnormality_queue.dm:84]: SSabnormality_queue.possible_abnormalities is a list of lists, it is supposed to double pick. Otherwise you end up with the value being... "TETH"

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/game/machinery/computer/abnormality_queue.dm (hunk 4 above)
Comment (Gboster-0) [reply code/game/machinery/computer/abnormality_queue.dm:84]: ah, gotcha. Reverting it

File: code/modules/mob/living/simple_animal/abnormality/waw/rose_sign.dm [hunk 5]
```diff
@@ -410,9 +428,24 @@
 			sin = "death"
 
 /obj/structure/rose_work/proc/RoseLines(mob/living/simple_animal/hostile/abnormality/rose_sign/master)
-	var/list/speech_styles = list(SPAN_ITALICS, SPAN_SANS, SPAN_ROBOT, SPAN_ITALICS, SPAN_PAPYRUS, SPAN_SANS, SPAN_SINGING)
-	var/list/lines = list("Ah, how pretty.", "And so vivid, too!", "Your sin was such a beautiful hue of [sin].", "You've really made a fine addition to the garden.", \
-	"The color [sin]?! I'll plant it right next to me.", "Then... Shall we play some more?", "This garden will become terribly beautiful with more sinful flowers we bloom!")
+	var/list/speech_styles = list(
+		SPAN_ITALICS,
```
Comment (Coxswain-Navigator) [inline code/modules/mob/living/simple_animal/abnormality/waw/rose_sign.dm:432]: This breaks rose sign's speech gimmick, which uses spans in a particular order

Comment (Gboster-0) [reply code/modules/mob/living/simple_animal/abnormality/waw/rose_sign.dm:433]: gotcha, changed dat

**Review by LanceSmites328 — APPROVED** [review]

**Review by LanceSmites328 — APPROVED** [review]

**Review by EgorDinamit — APPROVED** [review]
